WordPress/Classes/ViewRelated/Blog/QuickStartChecklistManager.swift

```diff
@@ -0,0 +1,178 @@
+class QuickStartChecklistManager: NSObject {
+    typealias QuickStartChecklistDidSelectTour = (String) -> Void
+    typealias QuickStartChecklistDidTapHeader = (Bool) -> Void
+
+    private var blog: Blog
+    private var tours: [QuickStartTour]
+    private var todoTours: [QuickStartTour] = []
+    private var completedTours: [QuickStartTour] = []
+    private var completedToursKeys = Set<String>()
+    private var didSelectTour: QuickStartChecklistDidSelectTour
+    private var didTapHeader: QuickStartChecklistDidTapHeader
+    private var completedSectionCollapse: Bool = false
+
+    init(blog: Blog,
+         tours: [QuickStartTour],
+         didSelectTour: @escaping QuickStartChecklistDidSelectTour,
+         didTapHeader: @escaping QuickStartChecklistDidTapHeader) {
+        self.blog = blog
+        self.tours = tours
+        self.didSelectTour = didSelectTour
+        self.didTapHeader = didTapHeader
+        super.init()
+        reloadData()
+    }
+
+    func reloadData() {
+        let completed = (blog.completedQuickStartTours ?? []).map { $0.tourID }
+        let skipped = (blog.skippedQuickStartTours ?? []).map { $0.tourID }
+
+        completedToursKeys = Set(completed).union(Set(skipped))
```

Comment:
I think there is a logic problem here. `blog.skippedQuickStartTours` are tours that have been skipped from the notices. This isn't supposed to mark them as complete. This is different than the `Skip` button(s) in the UI, which _do_ mark them as complete. That's a difference between what "skip" means in the code vs. the UI. 

In the new UI, the sideways UI will mark them as complete, as for all our purposes they will be. Let me know if that doesn't make sense.

Reply:
Could there be a benefit of making a distinction between skipped and completed for any future iterations we might make?